server/routes/barcode/PdfController.test.ts

```diff
@@ -109,4 +115,23 @@ describe('PdfController', () => {
       expect(res.redirect).toHaveBeenCalledWith('/barcode/pdf/select-envelope-size')
     })
   })
+
+  describe('submitPrintCoverSheet', () => {
```

Comment:
Need to write some more happy path tests here once a few questions on SLM-66 have been answered (I have some time booked with Liora and Lauren to discuss)